wordpress-mobile/WordPress-iOS
# Blaze: Show blaze overlay without post content

Part of #20094 

## Description
- Shows a blaze overlay modal from the dashboard card and menu item entry points

## Notes
- The following will be addressed in future PRs
  - UI refinement
  - Analytics 

## How to test

1. Switch to a Blaze eligible site
2. Switch to the "Home" tab
3. Tap on the Blaze dashboard card
4. ✅ A Blaze overlay should be displayed
5. Tap on "Blaze a post now"
6. ✅ A Blaze webview opens and you land in the posts lists screen
7. Close the webview
8. Switch to the "Menu" tab
9. Tap on the Blaze menu item
10. ✅ A Blaze overlay should be displayed
11. Tap on "Blaze a post now"
12. ✅ A Blaze webview opens and you land in the posts lists screen

## Regression Notes
1. Potential unintended areas of impact
n/a

13. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

14. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
